### PR TITLE
SYNBOT-TimeTracker: Adding new bot for timetracking.

### PR DESCRIPTION
**Pull request by santhoshgl**
New hubot  command to track the techdebt and performance.

>   hubot i worked on performance for 10 hours
>   hubot i worked on techdebt for 10 minutes
>   hubot report
>   hubot clear timetrack for all
>  hubot clear timetrack for @santhoshgl

### CODE REVIEW

File: scripts/timetracker.coffee [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#   hubot techdebt
+#
+# Author:
+#   Santhosh GL
+#
+module.exports = (robot) ->
+  # Command: i worked on performance for 10 hours.
+  robot.respond /i\s*worked\son+([\w\W]+)\sfor\s([\w\W]+)\s([\w\W]+)/i, (msg) ->
+    username = msg.message.user.name
+    taskType = msg.match[1].trim()
```
Comment (msound) [inline scripts/timetracker.coffee]: @santhoshgl do a `strtolower` on this before storing in database.

File: scripts/timetracker.coffee [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    time = msg.match[2].trim()
+    period = msg.match[3].trim()
```
Comment (msound) [inline scripts/timetracker.coffee]: @santhoshgl Consider doing a `strtolower` here also.

**Comment by santhoshgl** [comment]
screen shot 2018-03-19 at 7 42 40 pm

**Pull request merged**